### PR TITLE
Add translate to string

### PR DESCRIPTION
**Pull request by eileenmcnaughton**

### CODE REVIEW

File: cividiscount.php [hunk 1]
```diff
@@ -892,7 +892,7 @@ function _cividiscount_filter_membership_discounts($discounts, $membershipTypeVa
  * Calculate either a monetary or percentage discount.
  */
 function _cividiscount_calc_discount($amount, $label, $discount, $autodiscount, $currency = 'USD') {
-  $title = $autodiscount ? 'Includes Automatic Member Discount of' : "Includes Applied Discount Code {$discount['code']}";
+  $title = $autodiscount ? ts('Includes Automatic Member Discount of') : ts("Includes Applied Discount Code ") . $discount['code'];
```
Comment (colemanw) [inline cividiscount.php]: I'd say use a placeholder for that 2nd string.

Comment (eileenmcnaughton) [inline cividiscount.php]: Hmm - I've already lost interest in this one

**Comment by eileenmcnaughton** [comment]
OK - I logged as an issue instead https://github.com/dlobo/org.civicrm.module.cividiscount/issues/144

File: cividiscount.php (hunk 1 above)
Comment (colemanw) [inline cividiscount.php]: Ok I made the change.

Comment (eileenmcnaughton) [inline cividiscount.php]: Oh cool you did - I was too quick to assume you hadn't

**Pull request closed without merging**